### ISSUE TITLE
CKDatabaseOperations must be submitted to a CKDatabase

### ISSUE THREAD

**Issue by DSPerson** [caiyue1993/IceCream#201]
> 2020-06-28 11:17:13.920227+0800 Cliper[7490:124354] [xpc.exceptions] <NSXPCConnection: 0x60000300e5b0> connection to service on pid 667 named com.apple.cloudd: Exception caught during invocation of reply block to message 'fetchLongLivedOperationsWithIDs:setupInfo:completionHandler:'.
```
Ignored Exception: CKDatabaseOperations must be submitted to a CKDatabase
(
	0   CoreFoundation                      0x00007fff2d006d63 __exceptionPreprocess + 250
	1   libobjc.A.dylib                     0x00007fff62ef5bd4 objc_exception_throw + 48
	2   CloudKit                            0x00007fff2be70f8d -[CKContainer _scheduleConvenienceOperation:] + 0
	3   Cliper                              0x000000010003094f $s8IceCream15DatabaseManagerPAAE34resumeLongLivedOperationIfPossibleyyFySaySSGSg_s5Error_pSgtcfU_ySo11CKOperationCSg_AHtcfU_ + 895
	4   libswiftCloudKit.dylib              0x00007fff636a1b07 $sSo11CKOperationCSgs5Error_pSgIeggg_ACSo7NSErrorCSgIeyByy_TR + 71
	5   CloudKit                            0x00007fff2be78a11 __86-[CKContainer(ApplicationPermission) fetchLongLivedOperationWithID:completionHandler:]_block_invoke + 122
	6   CloudKit                            0x00007fff2be78723 __89-[CKContainer(ApplicationPermission) _fetchLongLivedOperationsWithIDs:completionHandler:]_block_invoke_2 + 747
	7   CoreFoundation                      0x00007fff2cf6502c __invoking___ + 140
	8   CoreFoundation                      0x00007fff2cf64ecf -[NSInvocation invoke] + 305
	9   Foundation                          0x00007fff2f5e29fd __NSXPCCONNECTION_IS_CALLING_OUT_TO_REPLY_BLOCK__ + 17
	10  Foundation                          0x00007fff2f5df700 -[NSXPCConnection _decodeAndInvokeReplyBlockWithEvent:sequence:replyInfo:] + 684
	11  Foundation                          0x00007fff2f8c0dc4 __88-[NSXPCConnection _sendInvocation:orArguments:count:methodSignature:selector:withProxy:]_block_invoke_3 + 216
	12  libxpc.dylib                        0x00007fff644a2004 _xpc_connection_reply_callout + 36
	13  libxpc.dylib                        0x00007fff644a1f8c _xpc_connection_call_reply_async + 69
	14  libdispatch.dylib                   0x0000000101f477d9 _dispatch_client_callout3 + 8
	15  libdispatch.dylib                   0x0000000101f66ac1 _dispatch_mach_msg_async_reply_invoke + 442
	16  libdispatch.dylib                   0x0000000101f4eb20 _dispatch_lane_serial_drain + 280
	17  libdispatch.dylib                   0x0000000101f4faf9 _dispatch_lane_invoke + 487
	18  libdispatch.dylib                   0x0000000101f5cf42 _dispatch_workloop_worker_thread + 681
	19  libsystem_pthread.dylib             0x0000000101fd3093 _pthread_wqthread + 290
	20  libsystem_pthread.dylib             0x0000000101fd2ef3 start_wqthread + 15
)
```

**Comment by DSPerson** [comment]
platform: macOS 10.15
save some CreamAssert

**Comment by kzhiquan** [comment]
same issue. macOS 10.15.5

**Comment by DSPerson** [comment]
> same issue. macOS 10.15.5

Yes; I used a temporary method； I don‘t use cocoapods; I used source code; this func ``` public func syncRecordsToCloudKit(recordsToStore: [CKRecord], recordIDsToDelete: [CKRecord.ID], completion: ((Error?) -> ())? = nil)``` , has property ```isLongLived``` default is true. I change it to false;

**Comment by caiyue1993** [comment]
Thank you guys. This issue is fixed in #247. Close this one.
